packages-exp/auth-exp/src/core/auth/auth_impl.test.ts

```diff
@@ -489,6 +489,17 @@ describe('core/auth/auth_impl useEmulator', () => {
       expect(normalEndpoint.calls.length).to.eq(0);
       expect(emulatorEndpoint.calls.length).to.eq(1);
     });
+
+    it('checks the scheme properly', () => {
+      expect(() => auth.useEmulator('http://localhost:2020')).not.to.throw;
+      delete auth.config.emulator;
+      expect(() => auth.useEmulator('https://localhost:2020')).not.to.throw;
+      delete auth.config.emulator;
+      expect(() => auth.useEmulator('ssh://localhost:2020')).to.throw(
```

Comment:
Let's add a test case for `localhost:5000` (with no `://`), a somewhat common misuse

Reply:
Done